### DISCUSSION TITLE
Overlay breaks when using mobile

### DISCUSSION THREAD

**Discussion by Bjottt** [community/community#13202] [Other Feature Feedback, Questions, & Ideas]
site overlay goes weird when in for you

**Comment by jfullstackdev** [comment]
can you send the screenshot of that ? 

but just to update this too

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
